docs/src/main/asciidoc/kubernetes-client.adoc

```diff
@@ -29,6 +29,39 @@ This will add the following to your pom.xml:
     </dependency>
 ----
 
+== Usage
+
+Quarkus configures a Bean of type `KubernetesClient` which can be injected into application code using the well known CDI methods.
+This client be can be configured with any using properties like in the following example:
```

Comment:
```suggestion
This client can be configured with any using properties like in the following example:
```
for one. But I don't understand the `any using properties`. `using` doesn't sound correct?

Reply:
Yeah, it's bad. I'll fix it